File: Programming Basics with Python/03 Conditional statements advanced/More exercises/04_Car_To_Go.py
budget = float(input())
season = input().lower()
car_class = ""
car_type = "Jeep"
car_price = 0
if budget <= 100:
    car_class = "Economy class"
    if season == "summer":
        car_price = budget * 0.35
        car_type = "Cabrio"
    else:
        car_price = budget * 0.65
elif 100 < budget <= 500:
    car_class = "Compact class"
    if season == "summer":
        car_price = budget * 0.45
        car_type = "Cabrio"
    else:
        car_price = budget * 0.80
else:
    car_class = "Luxury class"
    car_price = budget * 0.9
print(f"{car_class}\n{car_type} - {car_price:.2f}")

File: JS Basics/03 Nested Conditional Statements - More Exercises/04. Car To Go.js
function car(budget, season) {
    budget = Number(budget);
    let category = ""
    let type = ""
    let price = 0
    if (budget <= 100) {
        category = "Economy class";
        if (season == "Summer") {
            type = "Cabrio";
            price = budget * 0.35;
        }
        else {
            type = "Jeep";
            price = budget * 0.65;
        }
    }
    else if (100 < budget && budget <= 500) {
        category = "Compact class";
        if (season == "Summer") {
            type = "Cabrio";
            price = budget * 0.45;
        }
        else {
            type = "Jeep";
            price = budget * 0.8;
        }
    }
    else {
        category = "Luxury class";
        type = "Jeep";
        price = budget * 0.9;
    }
    console.log(category);
    console.log(`${type} - ${price.toFixed(2)}`)
}
car(99.99, "Summer")